ddtrace/contrib/aiohttp/patch.py

```diff
@@ -1,54 +1,127 @@
+import os
+
+from yarl import URL
+
 from ddtrace import config
 from ddtrace.internal.logger import get_logger
+from ddtrace.internal.utils import get_argument_value
+from ddtrace.internal.utils.formats import asbool
 from ddtrace.vendor import wrapt
 
-from ...internal.utils.wrappers import unwrap
+from ...ext import SpanTypes
 from ...pin import Pin
+from ...propagation.http import HTTPPropagator
+from ..async_trace_utils import with_traced_module
+from ..trace_utils import ext_service
+from ..trace_utils import set_http_meta
+from ..trace_utils import unwrap
+from ..trace_utils import wrap
 
 
 log = get_logger(__name__)
 
-try:
-    # instrument external packages only if they're available
-    import aiohttp_jinja2
+propagator = HTTPPropagator()
```

Comment:
why create an instance? they are just static methods

Reply:
yep, this is old code